wiki/Skinning/osu!taiko/id.md

```diff
@@ -0,0 +1,524 @@
+# Skinning osu!taiko
+
+Kamu dapat mengganti elemen permainan dari osu!taiko dengan menambahkan folder yang bernama `taiko` di dalam folder skin yang digunakan. Jika pengguna tidak melakukan ini, pengguna perlu mengaktifkan fitur ini melalui menu [options](/wiki/Options) (aktifkan opsi `Gunakan skin Taiko untuk mode Taiko`), jika tidak maka skin bawaan yang akan digunakan.
+
+## Pippidon
+
+`pippidonclear.png`
+
+![](img/pippidonclear.gif)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Ya][true] (lihat catatan) | ![Ya][true] | Normal | Pojok Kiri Bawah | - |
+
+Catatan:
+
+- Nama animasi: `pippidonclear{n}.png`.
+  - Kamu hanya dapat menganimasikan elemen ini dengan maksimum 7 *frame* (dari 0 sampai 6).
+  - Jika dianimasikan, sangat direkomendasikan untuk menganimasikan 7 *frame* tersebut. (Jika tidak, *frame* terakhir dari animasi akan digunakan untuk pengganti *frame* yang tidak dianimasikan dengan urutan *frame* di bawah ini.)
```

Comment:
```suggestion
  - Apabila dianimasikan, kamu sangat direkomendasikan untuk menganimasikan ketujuh *frame* tersebut. (Apabila tidak, *frame* terakhir dari animasi akan digunakan untuk pengganti *frame* yang tidak dianimasikan dengan urutan *frame* di bawah ini.)
```

Reply:
```suggestion
  - Apabila dianimasikan, kamu sangat direkomendasikan untuk menganimasikan ketujuh *frame* tersebut (Apabila tidak, *frame* terakhir dari animasi akan digunakan untuk menggantikan *frame* yang tidak dianimasikan dengan urutan *frame* di bawah ini).
```
I think this more fit the structure or this one

```suggestion
  - Apabila dianimasikan, kamu sangat direkomendasikan untuk menganimasikan ketujuh *frame* tersebut.
  - Apabila tidak dianimasikan, *frame* terakhir dari animasi akan digunakan untuk menggantikan *frame* yang tidak dianimasikan dengan urutan *frame* di bawah ini.
```

Reply:
yeah 👍